create-rust-app_cli/src/content/project.rs

```diff
@@ -80,12 +80,12 @@ fn add_bins_to_cargo_toml(project_dir: &std::path::PathBuf) -> Result<(), std::i
     let append_to_toml = format!(
         r#"
 [[bin]]
-name = "fullstack"
-path = ".cargo/bin/fullstack.rs"
+name = "run_yarn_fullstack"
+path = ".cargo/bin/run_yarn_fullstack.rs"
```

Comment:
is this rename necessary or were you trying to maintain the naming convention with tsync?
if it's not necessary, let's undo this change.